### PR TITLE
Move old WDL toil tests

### PR DESCRIPTION
**Pull request by stxue1**
This resolves [this issue](https://github.com/DataBiosphere/toil/issues/4507).

Added one extra test for [this issue](https://github.com/DataBiosphere/toil/issues/4513) and made some previous tests compatible with toil-wdl-runner.

### CODE REVIEW

File: format.py [hunk 1]
```diff
@@ -43,8 +46,25 @@ def yaml_sort(d):
...
+    elif isinstance(d, list):
+        for i, item in enumerate(d):
+            d[i] = yaml_sort_flow(item)
+
+    return d
+
 def main():
     yaml = ruamel.yaml.YAML()
+    yaml.preserve_quotes = True
+    # yaml.default_flow_style = None
```
Comment (adamnovak) [inline format.py:67]:
```suggestion
```

**Review by adamnovak — CHANGES_REQUESTED** [review]
I found one line of commented-out code.

But also I think that the system of having different copies of everything for the different WDL versions is starting to collapse. We're adding three complete and only slightly different copies of everything here, including the WDL files that need to have a version line at the top, but also including all the test inputs which are the same for all the versions of the tests.

If you change a test, how do you know which other copies of the test exist to change? If one of the three copies of one of these files was subtly wrong, how would I know?

I feel like we have two options:
* Reorganize the tests to be by test, and then within the test by WDL version. So the shared input file for the test and all the different WDL versions of it are in the same folder.
* And/or, keep just one copy of the test WDL, and generate the other WDL versions programmatically. Some tests might need manual rewriting for different WDL versions, or only be applicable to certain WDL versions, but mostly the same WDL file, except for the version lines, should produce the same output in all WDL versions. We could maybe by default set the right version on the first line, and if that isn't enough we could manually write a `.patch` file (or make it using `diff`) for that particular test, to convert it from a base version to each applicable WDL version with like the `patch` command.

How have you been maintaining the three copies of everything and keeping them straight so far? Do you know which tests have copies that are meaningfully different in the different versions?

File: format.py (hunk 1 above)
Comment (stxue1) [reply format.py:67]: Currently, I've been manually making sure the three copies produce the same output, but most of the tests should have the same WDL code with only the version number being the difference. I think most tests should not be too different between different versions, so I can try generating WDL tests programatically first.

**Comment by adamnovak** [comment]
I think we can merge this, but we really want to do something about the duplicated files or it will come back to bite us.

**Review by adamnovak — APPROVED** [review]

**Pull request merged**